spec/models/direct_file_data_spec.rb

```diff
@@ -528,23 +512,23 @@
   end
 
   describe '#spouse_is_a_dependent?' do
-    let(:xml) { StateFile::DirectFileApiResponseSampleService.new.read_xml("az_bert") }
+    let(:xml) { StateFile::DirectFileApiResponseSampleService.new.read_xml("nj_married_filing_jointly_spouse_claimed_dep") }
```

Comment:
Using a NJ fixture because none of our official ones currently have this scenario